source/pcic/client.py

```diff
@@ -6,17 +6,19 @@
 import json
 import re
 import io
-
+import o2x5xx
 
 class Client(object):
-    def __init__(self, address, port):
+    def __init__(self, address, port, timeout_insec = 3):
```

Comment:
Maybe you can use the timeout function from source/rpc/utils with an try except TimeoutError for the __init__ fct?

Reply:
@Kottanidisan see comment